radixdlt-core/radixdlt/src/main/java/com/radixdlt/environment/rx/ModuleRunnerImpl.java

```diff
@@ -103,23 +102,17 @@
   private final List<Subscription<?>> subscriptions;
   private final ImmutableList<Consumer<ScheduledExecutorService>> onStart;
 
-  private static class Subscription<T> {
-    final Observable<T> o;
-    final EventProcessor<T> p;
-
-    Subscription(Observable<T> o, EventProcessor<T> p) {
-      this.o = o;
-      this.p = p;
-    }
-
+  private record Subscription<T>(Observable<T> o, EventProcessor<T> p) {
     Disposable subscribe(Scheduler s) {
       return o.observeOn(s)
           .subscribe(
               p::process,
               e -> {
                 // TODO: Implement better error handling especially against Byzantine nodes.
```

Comment:
Can we remove the TODOs and add tasks in the backlog instead?

Reply:
- Adding task does not necessarily result to removing of TODO, as comment points exactly to the place where issue is present.
- Adding tickets for TODO's is a separate task, I just had no time for this.

Reply:
Doesn't sonar consider TODO a code smell?